SECURITY.md

```diff
@@ -0,0 +1,69 @@
+# Security Policy
+
+## Supported Versions
+
+We recommend to only use the most recent minor release.
+ 
+## Reporting a Vulnerability
+
+**Please do not file a public ticket** mentioning the vulnerability.
+
+To disclose a vulnerability, please submit a `Security Advisory` via the `Security` tab on the impacted repository.
+
+If a repository doesn't have the proper security reporting set up, please email `security@celestia.org` to report the vulnerability.
+ 
+## Disclosure Policy
+Vulnerability details may be shared with third parties after the vulnerability
+has been fixed and the program owner has provided permission to disclose or
+after 120 days from submission, whichever is sooner.
+
+### Scope
+There is nothing considered out-of-scope for testers and researchers following
+the rules outlined in this policy.
+
+### Rewards
+Celestia Labs has no formal reward policy and researchers should not expect a reward
+for discovering a vulnerability. Celestia Labs is nonetheless grateful for all
+legitimate discoveries of vulnerabilities, and is happy to acknowledge the
+vulnerability and the researchers after a fix has been widely deployed.
+
+
+### Official Communication Channel
+???
+
+
+### Comments on this Policy
+If you have suggestions on how this process could be improved please submit a merge request.
+
+### Expectation
+When working with us according to this policy, you can expect us to:
+
+- Extend Safe Harbor for your vulnerability research that is related to this policy;
+- Work with you to understand and validate your report, including a timely initial response to the submission;
+- Work to remediate discovered vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to report a unique vulnerability, and your report triggers a code or configuration change.
+
+### Ground Rules
+To encourage vulnerability research and to avoid any confusion between good-faith hacking and malicious attack, we ask that you:
+
+- Play by the rules. This includes following this policy, as well as any other relevant agreements. If there is any inconsistency between this policy and any other relevant terms, the terms of this policy will prevail;
+- Report any vulnerability you’ve discovered promptly;
+- Avoid violating the privacy of others, disrupting our systems, destroying data, and/or harming user experience;
+- Use only the Official Channels to discuss vulnerability information with us;
```

Comment:
Official Channels is undefined. Should that be "Official Communication Channels"?